README_database.md

```diff
@@ -0,0 +1,38 @@
+# Storing and retrieving results in the database
+
+## Initial Setup
+### Schema
+The initial schema is created with `utils/init_db.sql`. This will create the database, tables, and procedures.
```

Comment:
So this will not change anything with the existing DB, right?

Reply:
`utils/init_db.sql` will completely wipe the database (and data) and re-create all tables and procedures. For now, this is useful as the tables are changing frequently. E.g. if a column name changes, then it's more difficult to just re-name the column - we would need a migration script. In the future, we will instead have to individually add columns, etc. as desired, and any fundamental name changes would require some sort of migration script. Also, once permissions are set up properly, we can have a single admin user to make these changes so we don't accidentally wipe the database.
Something else I would like to do in the beginning is archive the imported CSV data, so if we need to restructure we can just re-import all data

Reply:
Correct we do not want someone by accident to wipe out the db.
Looks good.